store/storetest/team_store.go

```diff
@@ -1472,8 +1480,8 @@ func testResetAllTeamSchemes(t *testing.T, ss store.Store) {
 	res := <-ss.Team().ResetAllTeamSchemes()
 	assert.Nil(t, res.Err)
 
-	t1 = (<-ss.Team().Get(t1.Id)).Data.(*model.Team)
-	t2 = (<-ss.Team().Get(t2.Id)).Data.(*model.Team)
+	t1, _ = ss.Team().Get(t1.Id)
```

Comment:
add a `require.Nil(t, err)` for each call here to be sure we don't end up deferencing a nil pointer 2 lines later.